website/static/js/fangorn.js

```diff
@@ -1229,17 +1229,25 @@ var FGToolbar = {
         }
         //multiple selection icons
         if(items.length > 1 && ctrl.tb.multiselected()[0].data.provider !== 'github') {
-            generalButtons.push(
-                m.component(FGButton, {
-                    onclick: function() {
-                        var configOption = resolveconfigOption.call(ctrl.tb, item, 'removeEvent', [event, items]); // jshint ignore:line
-                        if(!configOption){ _removeEvent.call(ctrl.tb, null, items); }
-                    },
-                    tooltip: 'Delete all of the currently selected items.',
-                    icon: 'fa fa-trash',
-                    className : 'text-danger'
-                }, 'Delete Multiple')
-            );
+            var deleteList = [];
+            items.forEach(function(item, index, arr){
```

Comment:
There is no need to check every item; bail out early when the first item is found.

I'll make this change when I merge.